### DISCUSSION TITLE
Clever Cloud documentation with Hextra

### DISCUSSION THREAD

**Discussion by juliamrch** [imfing/hextra#259] [Show and tell]
We used Hextra to revamp our documentation website at Clever Cloud : https://developers.clever-cloud.com

We've spent time tweaking it a little bit and adding new features. It will be our pleasure to contribute by following your roadmap, this theme rocks!

**Comment by imfing** [comment]
Looks amazing! 🙌

Would you mind adding it to the [showcase](https://imfing.github.io/hextra/showcase/) page?


https://github.com/imfing/hextra/blob/09e9f52019edbfece778db2a493088e535d7676c/exampleSite/content/showcase/index.md

**Reply by juliamrch** [reply]
Done ✌️https://github.com/imfing/hextra/pull/262
